examples/hello-debian/Dockerfile

```diff
@@ -4,6 +4,8 @@ FROM debian:bookworm
 ARG VERSION=2.1.0
 # {{ x-release-please-end }}
 
+ARG CPP_SDK_VERSION=3.3.3
+
```

Comment:
In these dockerfiles, the `CPP_SDK_VERSION` arg is injected by CI. But since it's also hardcoded in the file (so that people can run these examples locally), we'd still have to remember to update these manually.

See PR body for a solution.